test/integrations/routes/articleRating.test.js

```diff
@@ -0,0 +1,34 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../../index';
+
+chai.use(chaiHttp);
+
+describe('Article Rating API endpoint', () => {
+  const token = process.env.JWT_TOKEN;
+  describe('/rating Post Endpoint', () => {
+    const rateArticle = {
+      stars: 5,
+    };
+
+    it('should create an article', async () => {
+      const res = await chai.request(app)
+        .post('/api/v1/articles/2/ratings')
+        .set({ Authorization: `Bearer ${token}` })
+        .send(rateArticle);
+      expect(res.body.code).to.equal(201);
+      expect(res.body.data).to.be.an('object');
+      expect(res.body.status).to.equal(true);
+    });
+  });
+
+  describe('/rating Get Endpoint', () => {
```

Comment:
#### Nitpick
- Can we test for some use-cases, for example, somebody trying to send value of 6 instead of 1- 5, edit rating...etc

Reply:
@aanchirinah yes we can. I will add more use-cases to the test